pkg/ops/iso.go

```diff
@@ -415,16 +424,25 @@ func (b BuildISOAction) createEFI(rootdir string, isoDir string) error {
 	b.cfg.Logger.Infof("Detected Flavor: %s", flavor)
 	if strings.Contains(strings.ToLower(flavor), "ubuntu") {
 		b.cfg.Logger.Infof("Ubuntu based ISO detected, copying grub.cfg to /EFI/ubuntu/grub.cfg")
-		err = utils.MkdirAll(b.cfg.Fs, filepath.Join(isoDir, "EFI/ubuntu/"), constants.DirPerm)
+		err = utils.MkdirAll(b.cfg.Fs, filepath.Join(temp, "EFI/ubuntu/"), constants.DirPerm)
```

Comment:
is this a mistake of mine? feels like it should still be isoDir, not sure

Also I guess this can be moved into the function

Reply:
no, its correct, we need to create that dir for sure in both places anyway. Isodir creation is just below

Reply:
I moved it into the function